build/Dockerfile

```diff
@@ -1,6 +1,16 @@
 FROM dstacademy/steamcmd:0.3
 MAINTAINER DST Academy <support@dst.academy>
 
+# Install dependencies.
```

Comment:
For what reasons has this been moved atop?

Reply:
If you change an argument you don't want to install the dependencies again. So they have to be atop.